src/styles/index.scss

```diff
@@ -1,3 +1,147 @@
 body {
-  margin: 0;
+  $card-border: #f3f3f3;
+  $color-black: #060b35;
+  $color-gray: #616070;
+  $color-blue: #00acdc;
+  $color-white: #fff;
+  $gap: 4px;
+  
+  body,
+  p,
+  a,
+  img,
+  div {
+    margin: 0;
+    box-sizing: border-box;
+    font-family: Roboto, Helvetica, sans-serif;
+  }
+  
+  .card {
+    width: 200px;
+    height: 406px;
```

Comment:
do not hardcode height. just set proper paddings and margins everywhere

Reply:
not fixed
```suggestion


```
![2023-10-03_13h31_00](https://github.com/mate-academy/layout_product-cards/assets/61904995/c17912e9-0322-4748-b23f-c9401074fd22)